scss/_variables.scss

```diff
@@ -1643,4 +1643,28 @@ $back-to-top-icon:               var(--#{$boosted-variable-prefix}chevron-icon)
 $back-to-top-icon-width:         add(.5rem, 1px) !default;
 $back-to-top-icon-height:        subtract(1rem, 1px) !default;
 // scss-docs-end back-to-top
+
+//// Stepped process
+// scss-docs-start stepped-process
+$stepped-process-max-items: 5 !default;
+$stepped-process-counter:   step !default; // Used as a counter name
+
+$step-item-margin-end:   $border-width !default;
+$step-item-padding:      $spacer / 2.5 !default;
```

Comment:
I'd go with `.5rem` instead of this: `$spacer / 2` is a value in our `$spacers` map, `/ 2.5` is not.

Reply:
I hesitated but ok, go for it (maybe should I store it into a variable too ?)